src/containers/ResetPassword/ResetPassword.js

```diff
@@ -0,0 +1,165 @@
+  const [newPassword, setNewPassword] = useState("");
+  const [confirmNewPassword, setConfirmNewPassword] = useState("");
+  const [error, setError] = useState(null);
+
+  const updateNewPassword = (e) => {
+    setNewPassword(e.target.value);
+  };
+  const updateConfirmNewPassword = (e) => {
+    setConfirmNewPassword(e.target.value);
+  };
+
+  const checkValues = () => {
+    if (newPassword === "" || confirmNewPassword === "") {
+      setError("CAN NOT BE EMPTY FIELD!");
+    } else if (newPassword !== confirmNewPassword) {
+      setError("MUST MATCH!");
+    } else if (newPassword === confirmNewPassword) {
+      setError("NEW PASSWORD HAS SET");
+    }
+    setNewPassword("");
+    setConfirmNewPassword("");
+  };
+
+  // it should return new password to backend
+  // it will be updated later on!
+  const onSubmit = () => {
+    return null;
+  };
+
+  const classes = useStyles();
+
+  return (
+    <Container component="main" maxWidth="sm">
+      <CssBaseline />
+      <div className={classes.resetPassword}>
+        <Avatar className={classes.avatar}>
+          <LockOutlinedIcon />
+        </Avatar>
+        <Typography component="h1" variant="h5">
+          Reset Password
+        </Typography>
+        <form onSubmit={onSubmit} className={classes.form}>
+          <TextField
+            fullWidth
+            margin="normal"
+            label="New Password"
+            name="password"
+            autoComplete="current-password"
+            id="outlined-password-input"
+            type="password"
+            variant="outlined"
+            value={newPassword}
+            onChange={updateNewPassword}
+          ></TextField>
+          <TextField
+            fullWidth
+            margin="normal"
+            label="Confirm New Password"
+            name="confirmPassword"
+            autoComplete="current-password"
+            id="outlined-password-input"
+            type="password"
+            variant="outlined"
+            value={confirmNewPassword}
+            onChange={updateConfirmNewPassword}
+          ></TextField>
+          <Box>
+            {error === "NEW PASSWORD HAS SET" ? (
+              <Typography
+                component="h2"
+                variant="body1"
+                className={classes.success}
+              >
+                {error}
+              </Typography>
+            ) : (
+              <Typography
+                component="h2"
+                variant="body1"
+                className={classes.error}
+              >
+                {error}
+              </Typography>
+            )}
+          </Box>
+          <Button
+            fullWidth
+            variant="contained"
+            color="primary"
+            className={classes.submit}
+            onClick={checkValues}
+          >
+            Submit
+          </Button>
+        </form>
+      </div>
+      <Box mt={8}>
+        <Copyright />
+      </Box>
+    </Container>
+  );
+}
+
+function Copyright() {
```

Comment:
Repeated Component.

Reply:
I made it on purpose, because you already change this component by removing the like it MUI site!, it will cause conflict when we merge, I gonna import once all the current PRs got merged